### ISSUE TITLE
Plan production of material

### ISSUE THREAD

**Issue by lauradereynal** [MozillaFoundation/research#117]
Goal: After the trips in Kenya and India, we gathered lots lots of material that needs to be produced into somethings. 
Can we go over what we have and continue production ? 
It would be great to upload some videos and sounds with our blog posts.

**Comment by lauradereynal** [comment]
https://docs.google.com/document/d/1GVbhxJgQr9eGjU6KWb95qo-PHaPhvWLPl2uvcEHtAu8/edit 
Here are our notes from India 

We should add Kenya's material in here too

**Comment by cassiemc** [comment]
Welcome back @LauraReynal and @secretrobotron. Would be fun to brainstorm diff. design solutions with someone from the design team, if that's helpful! This issue may be about something more direct/basic though.

**Comment by secretrobotron** [comment]
@cassiemc No, that would be super useful, actually. We have a bunch of ideas, and I'd love to bounce them off some designers to see how they can grow. Let @LauraReynal and I make a complete inventory of material and ideas first, and then brainstorming/help/fun would be great :+1:.

**Comment by cassiemc** [comment]
Cool! Just bring it to design crit some Wednesday or schedule a call, either way!
